metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Publication.java

```diff
@@ -66,8 +68,21 @@ public Long getId() {
     @JsonIgnore
     private LocalDate releaseDate;
 
+    /**
+     * Get the studyIds.
+     */
+    @Formula("(SELECT string_agg(concat(study.accession,'.',study.version),',') FROM study_publications " +
```

Comment:
Also, do we really want to have such a high granularity of control as to control studies based on their accession **and version?** I think just accession should be enough. For example imagine someone has access to a number of studies and then decides to roll out an updated version for several of them. This would necessitate updating all of the fields in Keycloak for every user which had access to those studies. Just access based on accession should be enough.

Reply:
okay that seems reasonable